_opcodes/op119-bgee.html

```diff
@@ -0,0 +1,28 @@
+---
+n: 119
+name: "Spell Effect: Mirror Image"
+param1: "Irrelevant"
+param2: "Irrelevant"
+bg1: 0
+bg2: 0
+bgee: 1
+iwd1: 0
+iwd2: 0
+pst: 0
+---
+Applies <a href="#op159">opcode 159</a> with a 'Parameter1' = 2 + ('Caster Level of the spell containing op119' / 3).<br />
+In particular:
+<ul>
+	<li>A Wizard spell with op119 will use the caster's Wizard Casting Level.</li>
+	<li>A Priest spell with op119 will use the caster's Priest Casting Level.</li>
+	<li>An Innate spell with op119 will use the caster's Innate Casting Level.</li>
+</ul>
+
+{% capture note %}
+A side note about non-spellcaster classes:<br /><br />
+
+A Mirror Image scroll used by a Thief with HLA Use Any Item will use the level 10 duration, but the level 1 Image count.<br />
+The scroll casts the spell at level 10, but the opcode checks the caster's Wizard Caster Level (which for a Thief is 1).
+{% endcapture %}
+
+{% include note.html %}
```

Comment:
This can be all added to the original. But please also keep its intro, since it's much more readable.